content/platform/data-services-cloud-console-api/home.md

```diff
@@ -0,0 +1,176 @@
+* **API Path** - Provides detail URL path as the end-point to issue the API call. Note that user must add the base path URL to extend this path to the correct resource end-point.
+* **API Parameter** - Provides the client to input information such as the object for the manipulation, select a filter to limit the returned objects, and other purposes.
+* **API Data/Body/Payload** - This the data passed along in different part of the REST API usually associated with operation such as POST/PATCH/PUT
+* **API Response** - Provides detail response information on the result of the particular API and may include more data in JSON format. 
+* **API Error Codes** - Provide result of the execution of the API, will return either good or error along with the error message due to incorrect, or unauthorized API call.
+
+### Supported API Categories (Services)
+
+The API categories for DSCC will grow in accordance to the expansion of the DSCC services that are going to released in future. Current services that are available, as recorded today:
+
+#### Common (Alletra-6K, Alletra-9K, Primera, Nimble)
+
+1. Authentication
+2. Tasks
+3. Event Audit
+4. AuthZ (User RBAC permissions)
+5. Issues
+6. controllers
+7. host-initiator-groups
+8. Host-initiators
+9. ports
+10. shelves
+11. storage-pools
+12. storage-systems
+13. system-settings
+14. volume-sets
+15. volumes
+
+#### Alletra-6K or Nimble
+
+1. protection-templates
+2. disks
+
+### Versioning
+
+The major version number will be provided in the resource path as "v1" in this example:
+
+```md
+/api/v1/<resource group>/...
+```
+
+Some examples of these resource paths that contain several resource groups under the same root:
+
+```markdown
+/api/v1/storage-systems/...
+
+/api/v1/controllers/...
+
+/api/v1/volumes/...
+```
+
+Existing clients will be able to maintain the backward compatibility from the higher major version incremental, and adopt newly introduced API.  However, both the new and old version of API will be supported until the announcement of the deprecation for the old version of API. Nonetheless, the older major version will always be frozen with exception of bug fixes. There will also be an announcement of deprecation in the header and sunset header. 
+
+```md
+/api/v1/<resource group>/...
+
+/api/v2/<resource group>/...
+```
+
+### HTTP Request Methods
+
+| HTTP Verbs | Description                                                         |
+| ---------- | ------------------------------------------------------------------- |
+| GET        | Retrieves target resource                                           |
+| POST       | Creates an entity or changes state                                  |
+| PUT        | Replaces target resource with data part of the HTTP Request payload |
+| DELETE     | Remove the target source                                            |
```

Comment:
Add an "s" to Remove so it is consistent with the other verbs above it; Retrieves, Creates, Replaces, Removes